installscripts/jazz-terraform-unix-noinstances/scripts/triggerJenkinsDeleteResources.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import argparse
+import subprocess
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--jenkins-url', help='Specify the Jenkins url')
+    parser.add_argument('--jenkins-username',
+                        help='Specify the Jenkins username')
+    parser.add_argument('--jenkins-password',
+                        help='Specify the Jenkins password')
+    args = parser.parse_args()
+    startJob(args)
+
+
+def startJob(args):
+    collectUserInput(args)
+    subprocess.check_call(
+        [
+            "curl",
+            "-sL",
+            ("http://%s/jnlpJars/jenkins-cli.jar") %
+            (args.jenkins_url),
+            "-o",
+            "jenkins-cli.jar"])
+    jenkins_build_command = \
+        ' java -jar jenkins-cli.jar -s http://%s '\
+        '-auth %s:%s build jenkins-delete-resources  -p input=all' \
```

Comment:
input = all is the default parameter. There is no provision to add specific account numbers

Reply:
updated with account arguments